### PR TITLE
configPath 

### PR DESCRIPTION
**Pull request by ewu02**
Sync with latest gulp-sass release where configPath can be specified

### CODE REVIEW

**Comment by alansouzati** [comment]
I really need this configPath option. That would be a great feature to add. I was about to write a PR for that, luckily you did this job already.

File: README.md [hunk 1]
```diff
@@ -23,3 +23,9 @@ gulp.task('default', function () {
     .pipe(sassLint.failOnError())
 });
 ```
+## Specify config file
+
+```
+  // Using 'sass-lint.yml' in cwd
+  .pipe(sassLint({}, 'sass-lint.yml'))
```
Comment (alansouzati) [inline README.md]: wouldn't be better to use options object for that?

Something like that:

```
.pipe(sassLint({ configPath: 'sass-lint.yml' }))
```

File: index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ var through = require('through2'),
 //////////////////////////////
 // Export
 //////////////////////////////
-var sassLint = function (options) {
+var sassLint = function (options, configPath) {
```
Comment (alansouzati) [inline index.js]: I believe we don't need an extra attribute to hold the configPath stuff.

**Comment by ewu02** [comment]
yea I like using the objects option for that as well. Made it as a separate argument because that's what gulp-sass did. 

I'll make the change shortly

**Comment by ewu02** [comment]
changes made

**Comment by Snugug** [comment]
I'm going to close this because I've opened https://github.com/sasstools/sass-lint/issues/226 to specifically address this in the core system and I'd prefer to handle it that way.

**Comment by Snugug** [comment]
Thank you for the PR though

**Pull request closed without merging**